app/views/transcripts/charts/_bubble_courses.html.haml

```diff
@@ -10,6 +18,7 @@
 
       // Set chart options
       var options = {
+        tooltip: { isHtml: true }, //It will make the chart tooltip be html rendered instead of svg rendered
```

Comment:
Hey @gustavosobral,
Nice to see you too. You guys have cool project here, I liked it.

I just finished the changes you requested. Sorry to mess with the build, not familiar with ruby x).